### ISSUE TITLE
documenting and understanding the API key

### ISSUE THREAD

**Issue by eroux** [OpenPecha/OCR-Pipelines#27]
Currently the API key field expects a json that contains a private key, etc. and is supposed to be generated through the following process:

https://user-images.githubusercontent.com/17675331/212290465-f7e3363f-97fd-4058-9fb4-c1c885fb8726.mp4

but this is counter-intuitive since API keys are usually just a string, not a json file. Perhaps there is a different type of API key we could use other than the JSON?

**Comment by eroux** [comment]
in any case, there should be a documentation page dedicated to this issue

**Comment by evanyerburgh** [comment]
I agree having a a JSON file instead of a key is unexpected. For now though, I've added a reference doc [here](https://openpecha.org/tools/ocr-pipeline-reference/#google-cloud-service-json-key-file) and noted this in the section on providing a key, and I also created a guide to getting a key [here](https://openpecha.org/tools/cloud-vision-key/). Hopefully this makes it clear enough, though the best would be to have a usual-style string API key
